utils/validators/validators.go

```diff
@@ -0,0 +1,108 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+// Package validators contains environment variables validator.
+package validators
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// MetricsResolutions contains standard Prometheus metrics resolutions.
+type MetricsResolutions struct {
+	HR time.Duration
+	MR time.Duration
+	LR time.Duration
+}
+
+// EnvSettings contains PMM Server settings.
```

Comment:
Something about environment in the comment?

Reply:
Added.